### DISCUSSION TITLE
Another multilingual problem

### DISCUSSION THREAD

**Discussion by tdlinton** [guoyingtao/Mantis#139] [Q&A]
Hi,

Thank you for putting this framework in GitHub. It looks very useful!

I am having sort of the inverse of the previously reported issue with localization in Mantis.

When I build the example program using Cocoa Pods it works fine. But when I build it using the Swift Package Manager instead, the internally used strings aren't being localized properly. So the Crop buttons show up as "Mantis.Cancel" and "Mantis.Done" instead of "Cancel" and "Done". Do you have any idea how to fix this? 

Things I've done to try to fix or debug it:
* Set CFBundleAllowMixedLocalizations to YES in info.plist
* Added a string localization file at the app level (as opposed to in the package). Strings in the app Views get localized properly (e.g., I can change a top level button's text using the strings file) but not those in Package Views (e.g., I still see "Mantis.Done" and "Mantis.Cancel").
* Tried deployment targets of iOS 15, 14, and 14.4
* Tried adding definitions of "Mantis.Done" and "Mantis.Cancel" in my app's top level strings file.

I'm using Xcode 13.1 with Swift language setting of 5.

Can you suggest anything else to try?

Thanks!
Tom

**Comment by guoyingtao** [comment]
You can use fix_SPM_localization_issue branch for the time being, it only shows English strings now. I will fix Swift Package Manager localization issue later.

**Reply by tdlinton** [reply]
Thank you for your quick reply! I'm trying to use the branch but so far have not been able to get it to build properly - I get an error saying the module Mantis is not recognized. Probably something I'm doing wrong. I'll keep working on it.
Tom

**Comment by guoyingtao** [comment]
@tdlinton I just released Mantis 1.7.5 which solved the  localization issue for SPM. You can use this new version instead of the branch `fix_SPM_localization_issue`

**Reply by tdlinton** [reply]
Thank you so much! It's working for me now!
